### ISSUE TITLE
1.1.58 java.lang.NoClassDefFoundError

### ISSUE THREAD

**Issue by minigrrl** [HL7/fhir-ig-publisher#223]
This only happens in 1.1.58 - not in 1.1.57:

```
Connect to Terminology Server at http://tx.fhir.org                              (00:14.0423)
Exception in thread "main" java.lang.NoClassDefFoundError: okhttp3/Request$Builder
at org.hl7.fhir.r5.utils.client.network.Client.issueGetResourceRequest(Client.java:89)
	at org.hl7.fhir.r5.utils.client.FHIRToolingClient.getCapabilitiesStatementQuick(FHIRToolingClient.java:155)
at org.hl7.fhir.convertors.txClient.TerminologyClientR5.getCapabilitiesStatementQuick(TerminologyClientR5.java:99)
at org.hl7.fhir.r5.context.SimpleWorkerContext.connectToTSServer(SimpleWorkerContext.java:298)
	at org.hl7.fhir.igtools.publisher.Publisher.initializeFromIg(Publisher.java:1873)
at org.hl7.fhir.igtools.publisher.Publisher.initialize(Publisher.java:1458)
	at org.hl7.fhir.igtools.publisher.Publisher.execute(Publisher.java:726)
at org.hl7.fhir.igtools.publisher.Publisher.main(Publisher.java:8417)
```
@markiantorno

**Comment by costateixeira** [comment]
This has been fixed with 5f1ee85

**Comment by markiantorno** [comment]
Fixed in 1.1.60.

Closing for now. Thanks for opening the ticket.
